packages/select-rich/src/LionSelectRich.js

```diff
@@ -219,6 +219,10 @@ export class LionSelectRich extends ScopedElementsMixin(
 
   firstUpdated(changedProperties) {
     super.firstUpdated(changedProperties);
+    if (this._listboxNode.childElementCount === 1) {
```

Comment:
It's a better practice to check for `this.formElements.length`, since we also allow for other visual elements inside the listbox node (think of separators/groups etc.)

Also, do we also update accordingly when new options are removed/added dynamically afterwards?